racer/Cargo.toml

```diff
@@ -16,7 +16,7 @@ path = "src/racer/lib.rs"
 bitflags = "1.0"
 log = "0.4"
 env_logger = "0.7.1"
-clap = "2.32"
+clap = "3.1.1"
```

Comment:
I don't think our fork/vendored copy of racer uses clap at all -- at least, I'm not seeing anything with a quick "rg racer".

So let's just drop the dependency?